cob_teleop/config/teleop_logitech.yaml

```diff
@@ -0,0 +1,97 @@
+#cob4-2
+##########
+# common params
+run_factor: 2.0
+joy_num_buttons: 12
+joy_num_axes: 6
+joy_num_modes: 4
+mode_switch_button: 8
+light_topic_name: /light_torso/light
+
+# axes
+axis_vx: 1
+axis_vy: 0
+axis_vz: 5
+axis_roll: 4
+axis_pitch: 3
+axis_yaw: 2
+
+# buttons
+deadman_button: 5
+safety_button: 4
+init_button: 9
+
+#mode1: Base
+run_button: 7 #is it a common parameter? should we use this button also for other twist controllers or velocity controllers?
+
+#mode2: Trajectory controller
+torso_sss_default_target_button: 1
+sensorring_sss_default_target_button: 3
+head_sss_default_target_button: 3
+arm_right_sss_default_target_button: 2
+arm_left_sss_default_target_button: 0
+
+#mode3: Velocity group controller
+torso_joint1_button: 0
+sensorring_joint1_button: 1
+head_joint1_button: 2
+arm_right_joint1_button: 0
+arm_right_joint2_button: 1
+arm_right_joint3_button: 2
+arm_right_joint4_button: 3
+arm_left_joint1_button: 0
+arm_left_joint2_button: 1
+arm_left_joint3_button: 2
+arm_left_joint4_button: 3
+
+right_indicator_button: 7
+left_indicator_button: 6
+up_down_button: 5
+right_left_button: 4
+
+#mode4: Twist controller
+torso_twist_button: 1
+head_twist_button: 3
+arm_right_twist_button: 2
+arm_left_twist_button: 0
+
+components: {
+  base: {
+    twist_topic_name: '/base/twist_controller/command',
```

Comment:
The `/base/` part could be removed and the `component` could be used to resolve the full topic_name.
(The value of `twist_topic_name` would then only be `twist_controller/command` - Note: no leading `/`)
This would reflect the insights we got from on-going work on multi-robot simulation!